tfhe-zk-pok/benches/pke_verify.rs

```diff
@@ -0,0 +1,57 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::rngs::StdRng;
+use rand::SeedableRng;
+use tfhe_zk_pok::proofs::pke::{prove, verify};
+use tfhe_zk_pok::proofs::ComputeLoad;
+use utils::{write_to_json, PKEV1_TEST_PARAMS, PKEV2_TEST_PARAMS};
+
+#[path = "./utils.rs"]
+mod utils;
+
+use crate::utils::init_params_v1;
+
+fn bench_pke_verify(c: &mut Criterion) {
+    let bench_shortname = "pke_zk_verify_v1";
+    let bench_name = format!("tfhe_zk_pok::{bench_shortname}");
+    let mut bench_group = c.benchmark_group(&bench_name);
+    bench_group
+        .sample_size(15)
+        .measurement_time(std::time::Duration::from_secs(60));
+
+    let rng = &mut StdRng::seed_from_u64(0);
+
+    for (params, param_name) in [
+        (PKEV1_TEST_PARAMS, "PKEV1_TEST_PARAMS"),
+        (PKEV2_TEST_PARAMS, "PKEV2_TEST_PARAMS"),
+    ] {
+        let (public_param, public_commit, private_commit, metadata) = init_params_v1(params);
+        let bits = params.k * 64;
```

Comment:
size ?

Reply:
we should not go above the zk capacity which is

k * effective_t.ilog2()

here you chose

k * 64 bits ?

Reply:
I think I misunderstood the code in `benches/integer/zk_pke.rs`, I'm fixing that !